### ISSUE TITLE
运行pipcook run pipeline-config.json提示错误

### ISSUE THREAD

**Issue by caoyonghao** [alibaba/pipcook#819]
按照demo例子配置了pipeline-config.json，运行的时候提示
`✖ run pipeline error: The shape of the input to "Flatten" is not fully defined (got ,,1024). Make sure to pass a complete "input_shape" or "batch_input_shape" argument to the first layer in your model.`
pipeline-config.json示例
https://alibaba.github.io/pipcook/#/zh-cn/manual/intro-to-pipeline
```
{
  "specVersion": "2.0",
  "datasource": "https://cdn.jsdelivr.net/gh/imgcook/pipcook-script@3cabbd6/scripts/image-classification-mobilenet/build/datasource.js?url=http://ai-sample.oss-cn-hangzhou.aliyuncs.com/image_classification/datasets/imageclass-test.zip",
  "dataflow": [
    "https://cdn.jsdelivr.net/gh/imgcook/pipcook-script@3cabbd6/scripts/image-classification-mobilenet/build/dataflow.js?size=224&size=224"
  ],
  "model": "https://cdn.jsdelivr.net/gh/imgcook/pipcook-script@3cabbd6/scripts/image-classification-mobilenet/build/model.js",
  "artifact": [{
    "processor": "pipcook-artifact-zip@0.0.2",
    "target": "/tmp/mobilenet-model.zip"
  }],
  "options": {
    "framework": "mobilenet@1.0.0",
    "train": {
      "epochs": 100,
      "validationRequired": true
    }
  }
}
```

**Comment by FeelyChau** [comment]
最新的PR #817 修复了这个问题，可以使用这个 Pipeline 文件重试看看:
It has been fixed in the latest PR #817 , you could try this pipeline again:
```json
{
  "specVersion": "2.0",
  "datasource": "https://cdn.jsdelivr.net/gh/imgcook/pipcook-script@7cdbe96/scripts/image-classification/build/datasource.js?url=http://ai-sample.oss-cn-hangzhou.aliyuncs.com/image_classification/datasets/imageclass-test.zip",
  "dataflow": [
    "https://cdn.jsdelivr.net/gh/imgcook/pipcook-script@7cdbe96/scripts/image-classification/build/dataflow.js?size=224&size=224"
  ],
  "model": "https://cdn.jsdelivr.net/gh/imgcook/pipcook-script@7cdbe96/scripts/image-classification/build/model.js",
  "options": {
    "framework": "mobilenet@1.0.0",
    "train": {
      "epochs": 10,
      "validationRequired": true
    }
  }
}
```

**Comment by caoyonghao** [comment]
按照新的配置已经解决
